public/static/locales/en/form-builder.json

```diff
@@ -185,6 +185,7 @@
   "settingsResponseHint1": "Provide a Government of Canada email address to send form responses to.",
   "settingsResponseHint2": "In order to publish, you need to provide a response delivery destination.",
   "settingsResponseNote": "Set response delivery before previewing delivery and publishing. <a href=\"/en/form-builder/support\" target=\"_blank\">Contact support</a>",
+  "settingsResponseNotePublished": "To change the response delivery destination. <a href=\"/en/form-builder/support\" target=\"_blank\">Contact support</a>",
```

Comment:
Is there a reason why we decided to include the HTML link in the translation file? It is the first time I am seeing this so I am wondering why we did not stick we our initial approach of having links in the code in between two translated strings.

Reply:
We've utilized this approach in a few spots i.e. `publishNoAuthText`

I will update but it's been helpful for larger chunks of text where links appear in the middle of chunks of text and also where text is in flux.